src/zelos/ext/platforms/linux/loader.py

```diff
@@ -214,7 +214,8 @@ def _stack_setup(self, thread):
         #   https://github.com/torvalds/linux/blob/v3.19/include/uapi/linux/auxvec.h
 
         module_path_ptr, str_len = self.memory.heap.allocstr(
-            f"/home/admin/{self.main_module_name}", alloc_name="Module Path"
+            f"{self._z.config.virtual_path}/{self._z.config.virtual_filename}",
```

Comment:
Virtual path may already include a forward slash, so you should instead use python's `posixpath.join` function to concat the `virtual_path` and `virtual_filename`

Reply:
done